src/app/settings/keyboard-mapping-settings/components/keyboard-mapping-settings-page/keyboard-mapping-settings-page.component.html

```diff
@@ -27,7 +27,7 @@
                     <sofie-rounded-box>{{ getShortcutName(keyboardMapping) }}</sofie-rounded-box>
                 </td>
                 <td>
-                    <sofie-rounded-box class="{{ keyboardMapping.action.metadata.contentType | actionColorClass }}">{{ keyboardMapping.action.name }}</sofie-rounded-box>
+                    <sofie-rounded-box *ngIf="keyboardMapping.action else invalidField"  class="{{ getKeyboardMappingActionContentType(keyboardMapping) | actionColorClass }}">{{ getKeyboardMappingActionName(keyboardMapping) }} </sofie-rounded-box>
```

Comment:
Seems like a lot of code for a single line to me

Reply:
This is no longer the case after changing the approach :-)